lib/configproxy.js

```diff
@@ -174,6 +175,7 @@ ConfigurableProxy.prototype.add_route = function (path, data) {
     this.routes[path] = data;
     this.trie.add(path, data);
     this.update_last_activity(path);
+    this.statsd.increment('route.add', 1);
```

Comment:
So either that default no-op needs to be moved to the ConfigProxy constructor, or an undefined/null statsd should be handled (I would guess handling null statsd is cheaper than calling no-op functions, but I'm not sure).